thirdparty/thorvg/patches/cpp20-fixes.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/thirdparty/thorvg/src/loaders/svg/tvgSvgSceneBuilder.cpp b/thirdparty/thorvg/src/loaders/svg/tvgSvgSceneBuilder.cpp
```

Comment:
Note that the changes in this patch are now upstreamed in thorvg[^1], so this would be removed when migrating to one of their upcoming releases (likely 1.0.0)

[^1]: thorvg/thorvg#3316